tree-sitter-markdown/src/scanner.c

```diff
@@ -0,0 +1,1446 @@
+#include <tree_sitter/parser.h>
+#include <wchar.h>
```

Comment:
```suggestion
#include <wchar.h>
#include <wctype.h>
#include <string.h>
```

string.h for memcpy, wctype.h for iswlower, iswalpha etc